src/Illuminate/Support/helpers.php

```diff
@@ -614,6 +614,23 @@ function object_get($object, $key, $default = null)
     }
 }
 
+if (! function_exists('preg_fetch')) {
+    /**
+     * Perform a regular expression match and returns the matches.
+     *
+     * @param  string  $pattern
+     * @param  string  $subject
+     * @param  int     $flags
+     * @return array
+     */
+    function preg_fetch($pattern, $subject, $flags = 0)
+    {
+        $result = preg_match($pattern, $subject, $matches, $flags);
+        
```

Comment:
I think StyleCI is telling you that you have spaces there and it wants to be a new empty line, but not sure